<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComponentDisplayerWidget</class>
 <widget class="QWidget" name="ComponentDisplayerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>150</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDisplayer" name="desisgnatorDisplayer" native="true"/>
      </item>
      <item>
       <widget class="QDisplayer" name="positionDisplayer" native="true"/>
      </item>
      <item>
       <widget class="QDisplayer" name="rotationDisplayer" native="true">
        <zorder>positionDisplayer</zorder>
       </widget>
      </item>
      <item>
       <widget class="ComponentParametersDisplayer" name="paramDisplayer" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="previousCompoButton">
     <property name="text">
      <string>Previous</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="nextCompoButton">
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="showListCB">
     <property name="text">
      <string>Show Components List</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QDisplayer</class>
   <extends>QWidget</extends>
   <header>qdisplayer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ComponentParametersDisplayer</class>
   <extends>QWidget</extends>
   <header>componentparametersdisplayer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
